Tests/UIKitTests/UITextFieldExtensionsTests.swift

```diff
@@ -145,6 +152,15 @@ final class UITextFieldExtensionsTests: XCTestCase {
         XCTAssertEqual(textfield.leftView?.frame.width, image.size.width + 5)
     }
 
+    func testAddPaddingImageRightIcon() {
+        let textfield = UITextField()
+        textfield.frame = CGRect(x: 0, y: 0, width: 100, height: 44)
+
+        let bundle = Bundle(for: UIImageExtensionsTests.self)
+        let image = UIImage(named: "TestImage", in: bundle)!
```

Comment:
My bad, I thought the parameter was optional.
```suggestion
        let image = UIImage(named: "TestImage", in: bundle, compatibleWith: nil)!
```